### PR TITLE
app-admin/kubelet: bump go-dockerclient for v1.10.x support

### PR DESCRIPTION
**Pull request by mischief**
the patches have moved to coreos/kubernetes release branch

### CODE REVIEW

**Comment by crawford** [comment]
LGTM

File: app-admin/kubelet/kubelet-1.1.2-r2.ebuild [hunk 1]
```diff
@@ -11,8 +11,9 @@ inherit flag-o-matic systemd toolchain-funcs
 DESCRIPTION="Kubernetes Container Manager"
 HOMEPAGE="http://kubernetes.io/"
 KEYWORDS="amd64"
-SRC_URI="https://github.com/GoogleCloudPlatform/kubernetes/archive/v${PV}.tar.gz -> ${P}.tar.gz"
-S="${WORKDIR}/kubernetes-${PV}"
+MY_PV="${PV/_p/_coreos.}"
+SRC_URI="https://github.com/coreos/kubernetes/archive/v${MY_PV}.tar.gz -> ${P}.tar.gz"
```
Comment (marineam) [inline app-admin/kubelet/kubelet-1.1.2-r2.ebuild]: No, this must use MY_P for the final file name, as is it is conflicting with the previous file name.

**Comment by mischief** [comment]
added explicit _p0 so that MY_PV expands correctly. ptal

File: app-admin/kubelet/Manifest [hunk 2]
```diff
@@ -0,0 +1 @@
+DIST kubelet-1.1.2.tar.gz 22384247 SHA256 ffbbf62d7fa324b6f4c3dcdb229e028204ec458f7f78fbf87856a72ab29ec942 SHA512 339fc40c75ea3c2ce22ec564715ace7ed50a01c87d0e4f8b86befc19c256c6f4d33ac4a46635a92ea721b7de9f6b597724b4f1eaa8f4ded60d3aff96cb179a76 WHIRLPOOL b9108f4c316ca9700d790c5e7e2b085e42ecd74a14500587d77d66f390b4883ec34c8795ae4e132c2e8c1352cb4c0f06caf5cb52d677413361c668c6472045bf
```
Comment (marineam) [inline app-admin/kubelet/Manifest]: Manifest needs a refresh, still referring to old conflicting name.

**Comment by marineam** [comment]
LGTM

**Pull request merged**